samples/msal-browser-samples/BrokerTestApp2.0/app/brokeredApp/auth.js

```diff
@@ -11,9 +13,12 @@ const isEdge = msedge > 0;
 let signInType;
 let username = "";
 
-// Create the main myMSALObj instance
-// configuration parameters are located at authConfig.js
+/*
+ * Create the main myMSALObj instance
+ * configuration parameters are located at authConfig.js
+ */
 const myMSALObj = new msal.PublicClientApplication(msalConfig);
+myMSALObj.initializeBrokering();
```

Comment:
We should show this being awaited or .thened.

Reply:
Agreed. Can't await on the top level so we'll need to use .then and probably enable the login buttons inside

Reply:
Done